gensim/models/utils_any2vec.py

```diff
@@ -192,7 +242,6 @@ def _load_word2vec_format(cls, fname, fvocab=None, binary=False, encoding='utf8'
             for line in fin:
                 word, count = utils.to_unicode(line, errors=unicode_errors).strip().split()
                 counts[word] = int(count)
-
```

Comment:
Keep the blank line please. It helps separate code blocks for readability.

Reply:
Corrected.